ushahidi/platform-client
# always allow admin to submit to surveys #3129

This pull request makes the following changes:
- Always selects and disables the Admin-options in settings->surveys->configure->who can contribute to this survey (if the specific-roles is selected)

Testing checklist:
- Log in as an admin
- [x] Create a new role and assign a new user with that role
- Go to settings-> surveys-> select a survey
- Click on configuration-tab
- Click on "Specific roles" in "Who can contribute to this survey
- [x] The admin-option should be greyed out and selected
- Select on the newly created role and save the survey
-  As an admin, try submitting a post to the survey
- [x] The post should save as normal
- Log out and login as the new member with the new role
- Submit a post to the survey
- [ ] The post should save as normal

Fixes ushahidi/platform#3129 .

Ping @ushahidi/platform
